data/registry/exporter-rust-user-events-logs.yml

```diff
@@ -0,0 +1,21 @@
+title: User Events Logs Exporter
```

Comment:
In the [kernel ](https://docs.kernel.org/trace/user_events.html) and [crate ](https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-user-events-logs) documentation, the term "User Events" is consistently referred to as user_events. We can stick with this notation.